README.md

```diff
@@ -20,23 +22,29 @@ There are two ways to download, install and run this Web Service:
 
  To run the app as part of the standard Companies House Vagrant set-up, follow [these set-up instructions](https://github.com/companieshouse/vagrant-chs-development-v2) on how to run Vagrant on your machine
 
- Once Vagrant is up and running, `ssh` into Vagrant by running:
+ Once Vagrant is up and running you can either start the service on its own:
+
  ```
-vagrant ssh chs-dev
+ ubic start chs.dev-hub.applications-developer-service
  ```
 
-Now run the `ubic status` command to see what services are running.
+ Or you start it up as part of the chs dev-hub group (you will need to do this in order to be able to use the api functionality properly):

```

Comment:
I think a full explanation is probably needed.
"This application relies on the static docs.developer app in order to sign in and the various backend APIs to perform data operations. Without these you will need to create a session before attempting to access the site as all pages require authorisation and there will be no Applications visible and you will be unable to create new ones.

Reply:
Changed